### PR TITLE
Start of an action to install kind,knative and sigstore pieces + tests.

### PR DESCRIPTION
**Pull request by vaikas**
Signed-off-by: Ville Aikas <vaikas@chainguard.dev>

#### Summary
Start of a github action that can be reused from other repos more easily.

#### Ticket Link

Fixes

#### Release Note

```release-note
Start of a github action reusable action.
```

### CODE REVIEW

**Comment by vaikas** [comment]
@cpanato I wasn't sure how to test this before actually merging it since it doesn't seem to be usable before it's in there. Any ideas? Looks like after it goes in, you can refer to the commit? Maybe I could try pointing it to my branch at that commit?

**Comment by vaikas** [comment]
@cpanato that seemed to work :) Well, I can run against it :)

File: .github/workflows/test-action.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      matrix:
+        k8s-version:
+        - v1.21.x
+
+        leg:
+        - test github action
+
+    steps:
+    - name: Test running the action
+      uses: vaikas/scaffolding/actions/setup@91b2092691e21133879a133e9c2243e1e1311fe9
```
Comment (cpanato) [inline .github/workflows/test-action.yaml:31]: to test it you can do the following

```
    steps:
    - uses: actions/checkout@a12a3943b4bdde767164f792f33f40b04645d846 # v3
    - name: Test running the action
      uses: ./actions/
```

Comment (cpanato) [reply .github/workflows/test-action.yaml:31]: see https://github.com/cpanato/testing-ci-providers/pull/72

**Comment by vaikas** [comment]
@cpanato awesomesauce, thanks! Trying with that now :)

**Review by cpanato — APPROVED** [review]
cool!!

**Pull request merged**